docs/administration-guide/databases/google-analytics.md

```diff
@@ -0,0 +1,76 @@
+
+
+## Metabase: Adding a Google Analytics Dataset
+
+Once you have created an **OAuth client ID** and obtained your **Account id** for your Google Analytics dataset, head over to your Metabase instance, click on the **settings** cog, and select **Admin** to bring up Admin mode. In the **Databases** section, click on the **Add database** button in the upper right.
+
+On the **ADD DATABASE** page, select **Google Analytics** from the **Database type** dropdown. Metabase will present you with the relevant configuration settings to fill out:
+
+![images](../images/google_analytics_add_database.png)
+
+### Settings
+
+#### Name
+
+**Name** is the title of your database in Metabase.
+
+#### Google Analytics Account ID
+
+Get the **Account id** from [Google Analytics Console](https://analytics.google.com/). Click the **Admin** cog, in the admin tab, go to the **Account Settings** section. You will find the **Account Id** below the Basic Settings heading.
+
+#### Client ID and Client Secret
+
+Paste the **Client ID** and **Client Secret** you created when following the [steps above](#google-cloud-platform-creating-oauth-client-id). These credentials should be correctly associated to scopes allowing interaction with Google Analytics API.
+
+#### Auth Code
+
+Once you provided **Client ID** and **Client Secret** with valid scopes, a **Click here to get an auth code** link will appear over the **Auth Code** text box.
+Authorize the connection with your Google Login Credentials to see the Auth Code. Copy the code and paste it in the text box.
+
+#### Automatically run queries when doing simple filtering and summarizing.
+
+_Default: Enabled_
+
+When this slider is on, Metabase will automatically run queries when users do simple explorations with the Summarize and Filter buttons when viewing a table or chart. You can turn this off if querying this database is slow. This setting doesn’t affect drill-throughs or SQL queries.
+
+#### This is a large database, so let me choose when Metabase syncs and scans
+
+_Default: Disabled_
+
+By default, Metabase does a lightweight hourly sync and an intensive daily scan of field values. If you have a large database, we recommend turning this on and reviewing when and how often the field value scans happen.
+
+### Save your database configuration
+
+When you're done, click the **Save** button. A modal should pop up, informing you that your database has been added.
+
+![Database added](../images/database-added.png)
+
+You can click on **Explore this data** to see some automatic explorations of your data, or click **I'm good thanks** to stay in the Admin Panel.
+
+Give Metabase some time to sync with your Google Analytics dataset, then exit Admin mode, click on **Browse Data**, find your database, and start exploring your data.
```

Comment:
```suggestion
Give Metabase some time to sync with your Google Analytics dataset, then exit Admin mode, click on **Browse Data**, find your database, and start exploring your data. Once Metabase is finished syncing, you should see the names of your Properties & Apps in the data browser. 
```

Reply:
GA is actually not synced, since all values are known and pre-defined, so they should be instantly available.

Reply:
Odd - when I did this locally, there was a period of maybe a minute where it was showing IDs instead of names and I was getting an error when attempting to query GA.

Reply:
Haven't tested for a while, but maybe something else is happening, or maybe I'm misunderstanding the sync+fingerprint code.
It still takes a while to scan, so that would be understandable it wasn't showing names until that was completed.

Reply:
Hi,thank you everyone for the suggestions and comments. Would it be ok then to say something like "wait for the initial scan to complete" instead of explaining the sync process?